wrappers/node/demo/faber.js

```diff
@@ -138,6 +138,9 @@ async function runFaber (options) {
       logger.info(`proofState=${proofProtocolState}`)
     }
 
+    logger.info('Deleting connection...')
+    await connectionToAlice.delete()
```

Comment:
Good idea to include this call, but `connectionToAlice` is still being used below in ` await vcxProof.getProof(connectionToAlice)`